build.rs

```diff
@@ -53,9 +53,13 @@ fn main() {
                .define("USE_FIELD_INV_BUILTIN", Some("1"))
                .define("USE_SCALAR_INV_BUILTIN", Some("1"))
                .define("ENABLE_MODULE_ECDH", Some("1"))
-               .define("USE_EXTERNAL_DEFAULT_CALLBACKS", Some("1"))
-               .define("ECMULT_WINDOW_SIZE", Some("15")); // This is the default in the configure file (`auto`)
+               .define("USE_EXTERNAL_DEFAULT_CALLBACKS", Some("1"));
 
+    if cfg!(feature = "lowmemory") {
+        base_config.define("ECMULT_WINDOW_SIZE", Some("3")); // This is the lowest accepted value by configure
```

Comment:
If we want anything > 2, then we shouldn't claim that this is the lowest accepted value.